### ISSUE TITLE
This question is about how to eliminate the white box in the image

### ISSUE THREAD

**Issue by mshmoon** [bodokaiser/piwise#21]
Every label of image have a white margin , I want to ignore the margin when I  train net ,But I do not know how to do,So i ask you this question Could you help me?

**Comment by bodokaiser** [comment]
You should be able to provide custom weights to the loss function so that you accumulate no gradients from the white margins.
